internal/osbuild/ostree_commit_assembler.go

```diff
@@ -0,0 +1,22 @@
+package osbuild
+
+// OSTreeCommitAssemblerOptions desrcibe how to assemble a tree into an OSTree commit.
+type OSTreeCommitAssemblerOptions struct {
+	Ref string                          `json:"ref"`
+	Tar OSTreeCommitAssemblerTarOptions `json:"tar"`
```

Comment:
The final version that landed in osbuild-15 also has `tmp-is-dir` and `parent` options. Not blocking, of course.

Reply:
Yeah, we only support the subset we need (in general). Happy to add if they are needed, of course.